### PR TITLE
feat: introduce installer

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ AUTHORS
 cover
 .coverage
 dist
+.mypy_cache
```
Comment (jd) [inline .gitignore:8]: Split in another PR

File: .gitignore [hunk 2]
```diff
@@ -15,3 +16,9 @@ prod.monitoring.env
 constraints.txt
 docs/node_modules
 docs/source/actions/action-list.html
+.git
+installer/node_modules
+installer/build
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (jd) [inline .gitignore:24]: Those last 3 should be in a different PR too

File: installer/asgi.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# under the License.
+
+import fastapi
+from fastapi.staticfiles import StaticFiles
+
+
+application = fastapi.FastAPI()
+application.mount(
+    "/", StaticFiles(directory="/app/installer/build", html=True), name="installer"
+)
```
Comment (jd) [inline installer/asgi.py:22]: Should this be placed in a installer directory so it's clear it's not the engine itself?

**Review by jd — CHANGES_REQUESTED** [review]
Is there a good reason to have this installer in this repo?

**Review by jd — COMMENTED** [review]
Do we need dependabot set up for this directory?

**Comment by sileht** [comment]
> Do we need dependabot set up for this directory?

yes

**Review by jd — APPROVED** [review]

**Pull request merged**